demos/map-list-toggle/index.php

```diff
@@ -8,8 +8,8 @@
 	<style>
 	.segmented-control { text-align:center; }
 	.segmented-control .ui-controlgroup { display:inline; margin:.2em 0px; }
-
-	#map-page { width:100%; height:100%; }
+	.map-page { width:100%; height:100%; }
```

Comment:
can you adjust this based on the css style guide please http://contribute.jquery.org/style-guide/css/